src/zenml/zen_server/deploy/helm/templates/server-db-job.yaml

```
@@ -0,0 +1,141 @@
+apiVersion: batch/v1
+kind: Job
+metadata:
+  name: {{ include "zenml.fullname" . }}-db-migration
+  labels:
+    {{- include "zenml.labels" . | nindent 4 }}
+  annotations:
+    "helm.sh/hook": pre-install,pre-upgrade
+    "helm.sh/hook-weight": "-1"
+    "helm.sh/hook-delete-policy": hook-succeeded
```

```suggestion
    "helm.sh/hook-delete-policy": before-hook-creation,hook-succeeded
```
This way, previous invocations of jobs that have failed are also cleaned up when the helm chart is re-applied.